certora/confs/Marketplace.conf

```diff
@@ -5,10 +5,10 @@
         "contracts/Groth16Verifier.sol",
         "certora/helpers/ERC20A.sol",
     ],
-    "parametric_contracts": ["Marketplace"],
+    "parametric_contracts": ["MarketplaceHarness"],
     "link" : [
-        "Marketplace:_token=ERC20A",
-        "Marketplace:_verifier=Groth16Verifier"
+        "MarketplaceHarness:_token=ERC20A",
+        "MarketplaceHarness:_verifier=Groth16Verifier"
     ],
     "msg": "Verifying Marketplace",
```

Comment:
Might want to say MarketplaceHarness here as well

Reply:
@0x-r4bbit I updated the message as well